_data/simple-icons.json

```diff
@@ -3610,6 +3610,11 @@
             "hex": "CC0000",
             "source": "https://github.com/jekyll/brand"
         },
+        {
+            "title": "Jellyfin",
+            "hex": "AA5CC3",
+            "source": "https://github.com/jellyfin/jellyfin-ux/blob/f4ab5b61e99a6b404255b285ad722cc5427eccf8/branding/SVG/icon-transparent-white.svg"
```

Comment:
We may be better linking to [this page](https://jellyfin.org/docs/general/contributing/branding.html#theme), given it includes details about colours as well as a link to the https://github.com/jellyfin/jellyfin-ux repo.